documentation/src/main/markdown/integrations/azureplugin/azurepluginreleasenotes.md

```diff
@@ -1,10 +1,22 @@
 # Release Notes for Azure DevOps Plugin
 
+## Version 10.0.0
+**Notice**
+
+The [company_name] Software Integrity Group is now [var_company_name]    
+If you are an existing [detect_product_short] user, manually uninstall the [company_name] [detect_product_short] Plugin and install this plugin to receive future updates.    
+If you are a new user, you may proceed with installing the [detect_product_long] Plugin.   
+
+**New features**
+
+* Plugin updated to support [detect_product_long] 10.   
+	* [detect_product_long] [Release Notes](../../currentreleasenotes.md)
+
 ## Version 9.0.0
```

Comment:
@sig-cpotts Here, it will be good to add 9.0.1 entry (which will be released on to old marketplace entry) and add something like - "If you are planning to use old deprecated plugin, it is essential to update to 9.0.1 before detect.synopsys.com and sig-repo.synopsys.com is decommissioned in Feb, 2025. Refer to https://community.blackduck.com/s/article/Black-Duck-Domain-Change-FAQ#SDInstru "

Reply:
@sig-cpotts Sorry I should have been more clear. What I meant by add 9.0.1 entry is release notes section for 9.0.1 (which as we discussed in the meeting, will be released on Old market place entry) in addition to overview.md change, @devmehtabd made some changes to add fallback logic to detect.blackduck.com which plugin 9.0.1 will start using. Like we did for core 9.10.1/8.11.2. that's what I meant to say. Dev can provide more info if needed. (The bullet point can stay)

Reply:
Ah, ok, gotcha. Thanks.